test/conformance/gatewayapi/gateway_conformance_test.go

```diff
@@ -31,7 +31,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/client/config"
 	"sigs.k8s.io/controller-runtime/pkg/log"
-	v1 "sigs.k8s.io/gateway-api/apis/v1"
+	gatewayapi_v1 "sigs.k8s.io/gateway-api/apis/v1"
 	"sigs.k8s.io/gateway-api/apis/v1alpha2"
 	"sigs.k8s.io/gateway-api/apis/v1beta1"
```

Comment:
Hmm, why aren't these being updated? Does it not add import aliases if they don't already exist?

Reply:
Yeah i thought these would have changed. There are some more settings in the `importas` linter that might be usable to force things, will take a look